tests/dataframe/test_sort.py

```diff
@@ -281,140 +281,129 @@ def test_sort_desc_nulls_first(make_df):
 
 
 @pytest.mark.parametrize(
-    "cast_to",
+    "desc,nulls_first,expected_data",
     [
-        DataType.float32(),
-        DataType.float64(),
-        DataType.int8(),
-        DataType.int16(),
-        DataType.int32(),
-        DataType.int64(),
-        DataType.uint8(),
-        DataType.uint16(),
-        DataType.uint32(),
-        DataType.uint64(),
+        (True, True, {"b": [None, None, 10, 5, 1]}),
+        (True, False, {"b": [10, 5, 1, None, None]}),
+        (False, True, {"b": [None, None, 1, 5, 10]}),
+        (False, False, {"b": [1, 5, 10, None, None]}),
     ],
 )
+def test_sort_combinations(desc, nulls_first, expected_data):
+    data = {"b": [1, 10, 5, None, None]}
+    df = daft.from_pydict(data)
+    actual = df.sort(by="b", desc=desc, nulls_first=nulls_first)
+    expected = daft.from_pydict(expected_data)
+    assert actual.to_pydict() == expected.to_pydict()
+
+
 @pytest.mark.parametrize(
-    "nulls_first,desc,expected",
+    "desc,nulls_first,expected",
     [
-        (
+        pytest.param(
             [False, False],
             [False, False],
-            {
-                "id1": [1, 1, 2, 2, None, None],
-                "id2": [2, None, 1, 2, 1, None],
-                "values": ["f1", "e1", "c1", "a1", "d1", "b1"],
-            },
+            {"a": [0, 1, -1, 10, None, None, 999], "id": [1, 2, 3, 3, 3, 4, None]},
+            id="desc desc, last last",
```

Comment:
Nit: the test IDs are flipped. We say `desc desc` but we want the values to be ascending.

Reply:
Same for the rest of the test cases